### ISSUE TITLE
Choice of pull requests

### ISSUE THREAD

**Issue by punkrokk** [mailgun/flanker#151]
How come some pull request get approved in less than three hours, and others are years old?

**Comment by horkhe** [comment]
The bitter truth is that those are Mailgun internal PRs.

tl;dr;

For very long time we have not paid enough... what the heck let's be honest - any attention to most of our open source repositories. I can dive into the reasons why, but that will probably be perceived by the community like whining, so I won't. But we do want to rectify the situation. So in the nearest future we (and I in particular) will be going through all the pending PRs. That does not mean that all of them are going to be accepted, but we will be considering them. I will start with updating them  asking authors if they are still interested and will proceed with those who responds.
